datasets/arabic_speech_corpus/README.md

```diff
@@ -123,31 +123,31 @@ An example from the dataset is:
 
 ### Curation Rationale
 
-[Needs More Information]
+Please refer to [PhD thesis](#Citation-Information).
```

Comment:
Would it be possible to summarize a bit the paper into this section if possible ? And then redirect people who want more details to the paper ? I think it would make it more practical for everyone, what do you think ?

(and same for the other sections)